thamos/config.py

```diff
@@ -46,6 +46,11 @@
 from .discover import discover_python_version
 from .discover import discover_platform
 from .discover import discover_base_image
+from .discover import discover_cudnn_version
```

Comment:
Hey, I feel like discover is cluttering the imports a lot. I think it is better to just
`import discover` and change all functions to `discover.<...>`